services/src/main/java/io/druid/cli/CliBroker.java

```diff
@@ -87,6 +88,7 @@ public void configure(Binder binder)
             binder.bind(CachingClusteredClient.class).in(LazySingleton.class);
             binder.bind(BrokerServerView.class).in(LazySingleton.class);
             binder.bind(TimelineServerView.class).to(BrokerServerView.class).in(LazySingleton.class);
+            binder.bind(CoordinatorClient.class).in(LazySingleton.class);
```

Comment:
the broker should be using its own internal cache, not calling to the coordinator